README.md

```diff
@@ -208,6 +213,183 @@ try {
 
 <a name="help"/>
 
+<a name="in-process-linking"/>
+
+## Trace in-process asynchronous execution
+
+You can use the SDK to trace asynchronous in-process code execution. This might be useful if Dynatrace does not support the threading framework or specific asynchronous libraries. In-process-linking will only be used to link other services (Database-, Webrequests, ...) between thread or queuing boundaries, where currently no support is available.
+
+To link asynchronous execution, you need to receive a InProcessLink, where async execution forks:
+
+```Java
+InProcessLink inProcessLink = OneAgentSDK.createInProcessLink();
+```
+
+The provided inProcessLink must not serialized and can only be used inside the process, where it has been created. It must be used to start tracing where the async execution takes place:
```

Comment:
inProcessLink should be `fixed font`
something is not correct in the wording here. In general I'm unsure if we should mention `serialize` at all here. In the end it's perfectly valid to serialize the link as long as it is deserialized in process after thread boundary.